### PR TITLE
fix redis startup + logfile location

### PR DESCRIPTION
**Pull request by grosser**
glueoutputbuf just blows up and logfile now needs to be "" to be stdout

@quirkey

### CODE REVIEW

**Comment by grosser** [comment]
added:
 - use minitest
 - use simpler exit hook
 - fix exit hook not killing redis because it is named `redis-server *:9763`

**Comment by grosser** [comment]
@quirkey please have a look here too :)

File: Gemfile [hunk 1]
```diff
@@ -4,7 +4,7 @@ gem 'resque', '~>1.19'
 
 group :test do
   gem 'mocha', '~>0.9'
-  gem 'shoulda', '~>2.10.2'
+  gem 'minitest'
```
Comment (quirkey) [inline Gemfile]: Can you change this to an explicit require of ~>5.5? Minitest < 5 has a different API

Comment (grosser) [inline Gemfile]: changed, same argument could be made for jeweler since version 0.1 had a different api etc ... so I think it's not necessary

**Comment by grosser** [comment]
rebased

**Comment by quirkey** [comment]
Thanks! Appreciate it.

**Pull request merged**